scripts/setenv.sh

```diff
@@ -1,10 +1,10 @@
 fn apps config set myapp PUBNUB_PUBLISH_KEY $PUBNUB_PUBLISH_KEY
 fn apps config set myapp PUBNUB_SUBSCRIBE_KEY $PUBNUB_SUBSCRIBE_KEY
-fn apps config set myapp FUNC_SERVER_URL ${API_URL}/r/myapp
+fn apps config set myapp FUNC_SERVER_URL ${FUNC_SERVER_URL}/r/myapp
 fn apps config set myapp MINIO_SERVER_URL $MINIO_SERVER_URL
+fn apps config set myapp ACCESS $MINIO_ACCESS_KEY
```

Comment:
it's actually not technically a Minio-specific keypair... the whole thing works with S3 as well with no code changes hence the names ACCESS/SECRET

Reply:
Oh, how about STORAGE_ACCESS_KEY or something then? access and secret aren't clear what it's for.

Reply:
ya that works. did you search the code for references to that ENV var? mind making those changes?